tests/test_manageiq_provider.py

```diff
@@ -304,7 +304,7 @@ def test_will_update_amazon_provider_if_present(miq, miq_api_class, amazon_endpo
         "other region")
     assert res_args == {
         'changed': True,
-        'msg': 'Successful update of {} provider. Authentication: All Valid. Refreshing provider inventory'.format(AMAZON_PROVIDER_NAME),
+        'msg': 'Successful update of {} provider. Authentication: {{}}. Refreshing provider inventory'.format(AMAZON_PROVIDER_NAME),
```

Comment:
The field we updated is `provider_region` (stored on ext_management_systems).
I think that will re-connect to another AWS endpoint?  https://docs.aws.amazon.com/general/latest/gr/rande.html
Will/should MIQ re-validate the auth in this case?
Anyway, if it's in 'Updated' dict, shouldn't it have been in `authtypes_to_verify`?